internal/cmd/commands/database/init.go

```diff
@@ -23,6 +23,20 @@ var (
 	_ cli.CommandAutocomplete = (*InitCommand)(nil)
 )
 
+type InitFlags struct {
+	flagConfig                                   []string
+	flagConfigKms                                string
+	flagLogLevel                                 string
+	flagLogFormat                                string
+	flagMigrationUrl                             string
+	flagSkipInitialLoginRoleCreation             bool
+	flagSkipInitialAuthenticatedUserRoleCreation bool
+	flagSkipAuthMethodCreation                   bool
+	flagSkipScopesCreation                       bool
+	flagSkipHostResourcesCreation                bool
+	flagSkipTargetCreation                       bool
+}
```

Comment:
Could you move the definition of this struct to live alongside the methods of the struct? Perhaps in a separate file?